themes/default/templates/nav.html

```diff
@@ -40,15 +40,14 @@ <h3 data-bind="text: name, typeset: name"></h3>
 
             <!-- stopwatch and timing controls -->
             <li id="stopWatch" class="fake-nav-link" data-bind="visible: displayTime">
-                <a class="clearfix">
+                <a class="clearfix">prev
                     <span class="pull-left" data-localise="timing.time remaining"></span>
                     <span class="pull-right" data-bind="text: displayTime"></span>
                 </a>
             </li>
 
             <div class="menuButtons">
                 <li><button class="btn btn-primary navbar-btn center-block" id="pauseBtn" data-localise="control.pause" data-bind="click: Numbas.controls.pauseExam, visible: allowPause && mode()=='normal'"></button></li>
-                <li><button class="btn btn-primary navbar-btn center-block" id="endBtn" data-localise="control.end exam" data-bind="click: Numbas.controls.endExam, visible: mode()=='normal'"></button></li>
```

Comment:
The existing End Exam button should stay where it is - the point of the issue is that there should be an **extra** button on the top nav bar.